fusor-ember-cli/app/controllers/register-nodes.js

```diff
@@ -466,13 +482,15 @@ export default Ember.Controller.extend({
       var checkForDone = function() {
         //ic-ajax request
         console.log('action: checkNodeIntrospection');
-        console.log('POST /fusor/api/openstack/deployments/' + self.get('deploymentId') + '/nodes/' + node.uuid + '/ready');
+        console.log('GET /fusor/api/openstack/deployments/' + self.get('deploymentId') + '/nodes/' + node.uuid + '/ready');
 
         request({
           url: '/fusor/api/openstack/deployments/' + self.get('deploymentId') + '/nodes/' + node.uuid + '/ready',
           type: 'GET',
           contentType: 'application/json'
         }).then(function(results) {
+            //refresh model
+            self.send('refreshNodesAndFlavors');
             resolve({done: results.node.ready});
```

Comment:
@sdherr, is there any case this would resolve successfully but `results.done.ready` is `false`, or is it always `true` if it resolves successfully.